portfolio/src/main/webapp/assets/js/script.js

```diff
@@ -99,4 +100,72 @@ function toggleImageGallery() {
     toggleButton.innerHTML = "view image gallery"
 
   }
+}
+
+function getComments() {
+    fetch("/data").then(response => response.json()).then((comments) => {
```

Comment:
consider renaming `/data` to `/comments` (this will require changes in other places too)